src/map/ai/controllers/mob_controller.cpp

```diff
@@ -531,7 +531,7 @@ void CMobController::CastSpell(SpellID spellid)
                 {
                     // chance to target party
                     PMob->PAI->TargetFind->reset();
-                    PMob->PAI->TargetFind->findWithinArea(PMob, AOE_RADIUS::ATTACKER, PSpell->getRange());
+                    PMob->PAI->TargetFind->findWithinArea(PMob, AOE_RADIUS::ATTACKER, PSpell->getRange(), 0, 0);
```

Comment:
Can we use the enum values here instead of 0s please?

FINDFLAGS_NONE etc

Reply:
there isnt an enum for targettype value 0

Reply:
```enum TARGETTYPE
{
    TARGET_SELF                    = 0x01,
    TARGET_PLAYER_PARTY            = 0x02,
    TARGET_ENEMY                   = 0x04,
    TARGET_PLAYER_ALLIANCE         = 0x08,
    TARGET_PLAYER                  = 0x10,
    TARGET_PLAYER_DEAD             = 0x20,
    TARGET_NPC                     = 0x40, // an npc is a mob that looks like an npc and fights on the side of the character
    TARGET_PLAYER_PARTY_PIANISSIMO = 0x80,
    TARGET_PET                     = 0x100,
    TARGET_PLAYER_PARTY_ENTRUST    = 0x200,
    TARGET_IGNORE_BATTLEID         = 0x400, // Can hit targets that do not have the same battle ID
};
```

Reply:
No reason you can't add `FINDFLAGS_NONE`, rather than piping in a raw number that means the same thing

Reply:
or `TARGET_NONE`, whatever

Reply:
Then the storage and the argument types become the enum:

```
    FIND_TYPE m_findType;
    FINDFLAGS    m_findFlags;   // what to search for
    TARGETTYPE    m_targetFlags; // targetflags to reject potentially bad targets
```

Reply:
Yeah, that one was a "we should probably do this at some point" thing, currently looks good 👍